weave/integrations/crewai/crewai_sdk.py

```diff
@@ -0,0 +1,180 @@
+from __future__ import annotations
```

Comment:
Maybe a short module docstring with links to CrewAI docs would be beneficial?

Reply:
added